spk/sonarr/src/service-setup.sh

```diff
@@ -30,7 +30,7 @@ fi
 LEGACY_CONFIG_DIR="${SYNOPKG_PKGDEST}/.config"
 
 # workaround for mono bug with armv5 (https://github.com/mono/mono/issues/12537)
-if [ "$SYNOPKG_DSM_ARCH" == "88f8621" -o "$SYNOPKG_DSM_ARCH" == "88f8622" ]; then
+if [ "$SYNOPKG_DSM_ARCH" == "88f8621" -o "$SYNOPKG_DSM_ARCH" == "88f8622" -o "$SYNOPKG_DSM_ARCH" == "88f6282" ]; then
```

Comment:
The correct fix would be  `[ "$SYNOPKG_DSM_ARCH" == "88f6281" -o "$SYNOPKG_DSM_ARCH" == "88f6282" ]`
This is for `88f628x` (synology) and `88f6281` (SynoCommunity) arches for models like DS213air, DS213, DS413j, DS112, DS112+, DS212, DS212+, RS212, RS812, DS212j, DS112j, and more.
"88f8621" and "88f8622" are invalid values.

Reply:
ah so it is currently wrong as well?